solidity/random-beacon/contracts/libraries/Heartbeat.sol

```diff
@@ -107,8 +109,8 @@ library Heartbeat {
             )
         ).toEthSignedMessageHash();
 
-        address[] memory groupMembers = sortitionPool.getIDOperators(
-            group.members
+        address[] memory selectedOperators = sortitionPool.getIDOperators(
```

Comment:
I would call them `groupMembersAddresses`. These are not operators that would be selected by the sortition pool right now and these could not be the operators selected by the sortition pool in the past in case someone misbehaved during DKG. We are calling sortition pool to return operator addresses based on their identifiers, that's all.